app/models/language.rb

```diff
@@ -67,7 +67,7 @@ def to_embedded_resource_h
   def to_resource_h
     as_json(only: %i(comment_type devicon editor_css_urls editor_html_urls editor_js_urls
                     extension feedback highlight_mode layout_css_urls layout_html_urls
-                    layout_js_urls name output_content_type prompt queriable runner_url
+                    layout_js_urls multifile name output_content_type prompt queriable runner_url
```

Comment:
We should add a migration, shoudn't we?

Reply:
I've added this migration in a recent commit.